recipes/libcugraph/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "libcugraph" %}
+{% set version = "0.19.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/rapidsai/cugraph/archive/refs/tags/v{{ version }}a.tar.gz
+    sha256: eea8eb788ee2e9f369b6d4911c8cb7c4819453e17aa855eac14af86ee27f627e
+  # libcugraph depends on the RMM header-only library of the same version.
+  - url: https://github.com/rapidsai/rmm/archive/refs/tags/v{{ version }}a.tar.gz
+    sha256: aa2b6542e51e9d3bec1c4053ab2a06af8fb8e9a94be3832418109904ebebb276
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version == "None")]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake>=3.12.4
+    - make
+    - boost-cpp>=1.66
+    - nccl>=2.8.4
+    - ucx-proc=*=gpu
+    - faiss-proc=*=cuda
+    - conda-forge::libfaiss=1.7.0
```

As these are linked to and are not build tools, would move them to `host`. Also shouldn't need to specify `libfaiss` coming from `conda-forge`

```suggestion
  host:
    - boost-cpp >=1.66
    - nccl >=2.8.4
    - ucx-proc=*=gpu
    - faiss-proc=*=cuda
    - libfaiss=1.7.0
```

---

Fixed in next commit.